build/csslint/xfail.list

```diff
@@ -93,7 +93,7 @@ apps/system/style/themes/default/buttons.css 3 0
 apps/system/style/themes/default/core.css 4 5
 apps/system/style/ttlview/ttlview.css 1 0
 apps/system/style/update_manager/update_manager.css 4 0
-apps/system/style/utility_tray/utility_tray.css 3 0
+apps/system/style/utility_tray/utility_tray.css 4 0
```

Comment:
out of curiosity, what is bugging the linter? the var itself? changing it's value?